math/src/field/smallprimefield/mod.rs

```diff
@@ -0,0 +1,26 @@
+use core::convert::TryInto;
+
+use crate::StarkField;
+
+pub mod generic_prime_field_elt;
```

Comment:
I'm not sure we need this extra level of module nesting. Could we put the code from `generic_prime_field_elt.rs` into this file directly?

Reply:
Yes, thanks!